tests/integration/common/auth/test_auth.py

```diff
@@ -163,3 +163,10 @@ def test_get(self, anonymous_client, factories):
 
         with anonymous_client.session_transaction() as session:
             assert "_user_id" not in session
+
+
+class TestSSOSignInView:
+    def test_get(self, anonymous_client):
+        response = anonymous_client.get(url_for("auth.sso_sign_in"))
+        assert response.status_code == 200
+        assert b"Connected and consistent set of funding services" in response.data
```

Comment:
We could add a 🤏  test for the POST endpoint with `follow_redirects=False` to assert that the redirect goes to our configured redirect place?